packages/react-button/src/components/Button/stories/ButtonDisabled.stories.tsx

```diff
@@ -31,7 +31,13 @@ Disabled.parameters = {
   docs: {
     description: {
       story: `A button can be \`disabled\` or \`disabledFocusable\`.
-              \`disabledFocusable\` is used in scenarios where it is important to keep a consistent tab order
+      \`disabledFocusable\` is used when discoverability
+      of a disabled button using the keyboard and screen reader is important, i.e.,
+      when the presence of such button cannot be implied from the presence of another button.
+      For example, a "Increase font size" button implies the presence of a "Decrese font size" button,
+      but the presence of a disabled "Send message" button is not implied by the presence of any other button,
+      thus it should be \`disabledFocusable\`.
```

Comment:
I wonder if we should update the stories to include a bit more context on why a button would be disabled vs. disabledFocusable, since understanding the difference just be reading a text description of examples is a little difficult.

I'm even having some trouble understanding this recommendation without seeing an example, since it seems the reverse of what I would have expected in both cases:
- A disabled "send message" button sounds like a disabled submit button to me, without more context. I'd expect the recommendation there to be A) don't do that, and B) if doing that, don't put it in the tab order.
- Increase/decrease font size sound like the sort of buttons that would exist within a menu or menubar, and I would actually expect disabled menu items to remain focusable by arrow keys. Maybe a previous page/next page set might work better?

Aside from providing concrete interactive examples, I'm not sure I fully agree with the advice that a disabled button should be tabbable or not depending on the context of other buttons. Generally I think good baseline advice is that a control should not be disabled unless its existence is fully irrelevant to the user at the current moment (and usually in those cases, it's better to remove it entirely). The exception is usually where consistent tab/arrow order is important, so mostly in compound widgets like menu, tabs, combobox, etc.